### PR TITLE
Fix #60: introduce 'ignored_error_codes'

### LINKED ISSUE #60: NotificationFailed flow not ideal
When a queued SMS notification fails to send, a NotificationFailed event is fired. Currently, the NotificationSent event is still fired and all looks green in Telescope. Could the TwilioChannel be made a bit more verbose and mark the queued job as failed?

Aside: Would be nice if the documentation would cover catching NotificationFailed events.

### PR DESCRIPTION
**Pull request by klimov-paul**
Fixes #60. 
Config option 'ignored_error_codes' introduced, allowing normal processing of any other exception.

### CODE REVIEW

File: src/TwilioConfig.php [hunk 1]
```diff
@@ -92,4 +94,21 @@ public function getServiceSid()
...
+     * Returns list of codes for Twilio errors, which should be ignored (suppressed).
+     *
+     * @see https://www.twilio.com/docs/api/errors
+     *
+     * @return array
+     */
+    public function getIgnoredErrorCodes()
+    {
+        return Arr::get($this->config, 'ignored_error_codes', [
+            21608,
```
Comment (atymic) [inline src/TwilioConfig.php:108]: Could we set these defaults in the config file (and add comments to clarify what each one is)?

File: src/Twilio.php [hunk 2]
```diff
@@ -34,22 +35,31 @@ public function __construct(TwilioService $twilioService, TwilioConfig $config)
...
+                return $this->sendSmsMessage($message, $to);
             }
 
-            return $this->sendSmsMessage($message, $to);
-        }
+            if ($message instanceof TwilioCallMessage) {
+                return $this->makeCall($message, $to);
+            }
+        } catch (RestException $e) {
+            // suppress ignored errors :
```
Comment (atymic) [inline src/Twilio.php:57]: Not sure this comment is needed :)

**Comment by atymic** [comment]
See: https://github.com/laravel-notification-channels/channels/issues/16

File: src/TwilioConfig.php (hunk 1 above)
Comment (klimov-paul) [reply src/TwilioConfig.php:108]: This extension does not provide pre-defined config file.
Developer should manually add 'twilio' section into 'config/services.php'.
Thus such solution is preferrable.

You can find all error codes and thier meaning at https://www.twilio.com/docs/api/errors, which is linked in PHPDoc here.

Comment (atymic) [reply src/TwilioConfig.php:108]: I think if we're going with config this complex, we should move to a dedicated config file.
Will require a major version bump (unless we implement some kind of fallback to the old services behavior), but I think it's worth it.

Comment (atymic) [reply src/TwilioConfig.php:108]: I think we should also be documenting this in the readme as well.

Comment (klimov-paul) [reply src/TwilioConfig.php:108]: In case you are going to make an overhauling for whole package, there is no sense in polishing this particular PR.

**Comment by gregoriohc** [comment]
I'm closing this in favor of #90

**Pull request closed without merging**